sqlwrapper/ghostferry_db.go

```diff
@@ -90,5 +91,5 @@ func (tx Tx) QueryRow(query string, args ...interface{}) *sqlorig.Row {
 }
 
 func Annotate(query, marginalia string) string {
-	return marginalia + query
+	return fmt.Sprintf("/*%s*/ %s", marginalia, query)
```

Comment:
This is a breaking change.. right? I'd avoid it I think

Reply:
Changed because we're not asking the user to provide the `/*` and `*/` in the configured annotation. 

It would only break something if the provided annotation happened to also have the above, ie: `/*myannotation*/` as opposed to just `myannotation`.

If we're really concerned I can add a check for a "valid" annotation in the `ValidateConfig` to make sure the user isn't providing a comment char...what do you think?

Reply:
I'm not concerned about user input tbh.. we just have to remember to upgrade the shop mover side so the current config doesn't result in `/*/* stuff */*/`

Reply:
it's a valid change.. lets keep it 👍